src/components/ArticleView/ArticleCard.jsx

```diff
@@ -0,0 +1,13 @@
+const ArticleCard = ()=>{
```

Comment:
not sure what this component is supposed to be doing right now - try to make sure you only have code within each branch that is relevant to that specific ticket